Source/WebKit/UIProcess/Inspector/WebInspectorUIProxy.cpp

```diff
@@ -78,7 +78,7 @@ WebInspectorUIProxy::WebInspectorUIProxy(WebPageProxy& inspectedPage)
     , m_closeFrontendAfterInactivityTimer(RunLoop::main(), this, &WebInspectorUIProxy::closeFrontendAfterInactivityTimerFired)
 #endif
 {
-    m_inspectedPage->protectedLegacyMainFrameProcess()->addMessageReceiver(Messages::WebInspectorUIProxy::messageReceiverName(), m_inspectedPage->webPageIDInMainFrameProcess(), *this);
+    protectedInspectedPage()->protectedLegacyMainFrameProcess()->addMessageReceiver(Messages::WebInspectorUIProxy::messageReceiverName(), m_inspectedPage->webPageIDInMainFrameProcess(), *this);
```

Comment:
This is fine as is but I find it somehow a bit odd that we use protectedInspectedPage() and m_inspectedPage. Ditto elsewhere.
Maybe a local variable would make thinks more readable.

Reply:
Fixed to use a local variable.